docs/docs/fundamentals/animations.md

```diff
@@ -381,3 +381,24 @@ For this purpose Reanimated exposes a separate hook called `useAnimatedProps`.
 It works in a very similar way to `useAnimatedStyle` but instead of expecting a method that returns the animated styles, we expect the returned object to contain properties that we want to animate.
 Then, in order to hook animated props to a view, we provide the resulting object as `animatedProps` property to the "Animated" version of the view type we want to render (e.g. `Animated.View`).
 Please check the documentation of the [`useAnimatedProps`](../api/hooks/useAnimatedProps) hook for usage examples.
+
+## Shared Values in Properties
+You can also pass a shared value as a property like this:
+```js
+const AnimatedCircle = Animated.createAnimatedComponent(Circle);
+
+Animated.addWhitelistedNativeProps({ r: true });
```

Comment:
Can we remove this line now? Also, there is similar example in the PR description